brainiak/fcma/cython_blas.pyxbld

```diff
@@ -0,0 +1,13 @@
+def make_ext(modname, pyxfilename):
+    from distutils.extension import Extension
+    from distutils import sysconfig
+    import os
+    opts = []
+    if ('CC' in os.environ and 'icc' in os.environ['CC']) or \
+        'icc' in sysconfig.get_config_var('CC'):
+            opts += ['-lirc', '-lintlc']
```

Comment:
I looked for documentation on these libraries, but cannot find anything. Why do we need to add them explicitly?